{"uid":"1f7ae3c832eb1eeb","name":"test_precision[Test for KILT]","fullName":"tests.test_network_parameters.TestPrecision#test_precision","historyId":"d18900cd28188dad91c5f89dd9271bda","time":{"start":1669653148135,"stop":1669653148768,"duration":633},"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","statusTrace":"self = <tests.test_network_parameters.TestPrecision object at 0x7efcd42cd190>\nchain = <tests.utils.chain_model.Chain object at 0x7efcd43420d0>\n\n    def test_precision(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7efcd43420d0>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] KILT\n\ntests/utils/chain_model.py:29: TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","statusTrace":"self = <tests.test_network_parameters.TestPrecision object at 0x7efcd42cd190>\nchain = <tests.utils.chain_model.Chain object at 0x7efcd43420d0>\n\n    def test_precision(self, chain: Chain):\n>       chain.create_connection()\n\ntests/test_network_parameters.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <tests.utils.chain_model.Chain object at 0x7efcd43420d0>\n\n    def create_connection(self):\n        for node in self.nodes:\n            if (self.substrate):\n                return self.substrate\n            try:\n                self.substrate = create_connection_by_url(node.get('url'))\n                return self.substrate\n            except:\n                print(\"Can't connect to that node\")\n                continue\n    \n>       raise TimeoutError(\"Can't connect to all nodes of network\", self.name)\nE       TimeoutError: [Errno Can't connect to all nodes of network] KILT\n\ntests/utils/chain_model.py:29: TimeoutError","steps":[],"attachments":[{"uid":"417909bebc61316a","name":"stdout","source":"417909bebc61316a.txt","type":"text/plain","size":108}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_network_parameters"},{"name":"subSuite","value":"TestPrecision"},{"name":"host","value":"fv-az177-728"},{"name":"thread","value":"1613-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_network_parameters"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"chain","value":"<tests.utils.chain_model.Chain object at 0x7efcd43420d0>"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"8ee798f63cce1e31","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669653137433,"stop":1669653138118,"duration":685}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":20,"skipped":0,"passed":147,"unknown":0,"total":167},"items":[{"uid":"8c8769f6f4247e58","reportUrl":"https://nova-wallet.github.io/test-runner/1983//#testresult/8c8769f6f4247e58","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669639496443,"stop":1669639497573,"duration":1130}},{"uid":"433f96d47e2665f7","reportUrl":"https://nova-wallet.github.io/test-runner/1982//#testresult/433f96d47e2665f7","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669624471308,"stop":1669624471910,"duration":602}},{"uid":"3f6cee71040fdd07","reportUrl":"https://nova-wallet.github.io/test-runner/1981//#testresult/3f6cee71040fdd07","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669610006322,"stop":1669610006903,"duration":581}},{"uid":"b8301cb13e86eaba","reportUrl":"https://nova-wallet.github.io/test-runner/1980//#testresult/b8301cb13e86eaba","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669598719680,"stop":1669598720273,"duration":593}},{"uid":"6e25bb2adbf140ed","reportUrl":"https://nova-wallet.github.io/test-runner/1979//#testresult/6e25bb2adbf140ed","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669581016272,"stop":1669581017100,"duration":828}},{"uid":"21d910247ad7d85f","reportUrl":"https://nova-wallet.github.io/test-runner/1978//#testresult/21d910247ad7d85f","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669566972897,"stop":1669566974066,"duration":1169}},{"uid":"d0c5e3d8ea6dd57f","reportUrl":"https://nova-wallet.github.io/test-runner/1977//#testresult/d0c5e3d8ea6dd57f","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669552839952,"stop":1669552840547,"duration":595}},{"uid":"99befb9ac45b036a","reportUrl":"https://nova-wallet.github.io/test-runner/1976//#testresult/99befb9ac45b036a","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669537994253,"stop":1669537995051,"duration":798}},{"uid":"682d250ccbb9d6ce","reportUrl":"https://nova-wallet.github.io/test-runner/1975//#testresult/682d250ccbb9d6ce","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669523589829,"stop":1669523590700,"duration":871}},{"uid":"b26ad7aec4497199","reportUrl":"https://nova-wallet.github.io/test-runner/1973//#testresult/b26ad7aec4497199","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669494049742,"stop":1669494050597,"duration":855}},{"uid":"926fd8490bbf9c52","reportUrl":"https://nova-wallet.github.io/test-runner/1972//#testresult/926fd8490bbf9c52","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669479690528,"stop":1669479691400,"duration":872}},{"uid":"a18c09212d86af9a","reportUrl":"https://nova-wallet.github.io/test-runner/1971//#testresult/a18c09212d86af9a","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669465321106,"stop":1669465321930,"duration":824}},{"uid":"bc9f60b272533581","reportUrl":"https://nova-wallet.github.io/test-runner/1970//#testresult/bc9f60b272533581","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669450839427,"stop":1669450840264,"duration":837}},{"uid":"4921ff915817d5c5","reportUrl":"https://nova-wallet.github.io/test-runner/1969//#testresult/4921ff915817d5c5","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669436460341,"stop":1669436461476,"duration":1135}},{"uid":"5f3c887ec5a5a359","reportUrl":"https://nova-wallet.github.io/test-runner/1968//#testresult/5f3c887ec5a5a359","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669422510178,"stop":1669422510765,"duration":587}},{"uid":"c3496421f9e57e87","reportUrl":"https://nova-wallet.github.io/test-runner/1967//#testresult/c3496421f9e57e87","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669407598551,"stop":1669407599364,"duration":813}},{"uid":"f5ab815317956998","reportUrl":"https://nova-wallet.github.io/test-runner/1966//#testresult/f5ab815317956998","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669393205343,"stop":1669393206191,"duration":848}},{"uid":"f1b8ad12bff92cac","reportUrl":"https://nova-wallet.github.io/test-runner/1965//#testresult/f1b8ad12bff92cac","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669378854027,"stop":1669378854881,"duration":854}},{"uid":"d0d42cee9fdcec28","reportUrl":"https://nova-wallet.github.io/test-runner/1964//#testresult/d0d42cee9fdcec28","status":"broken","statusDetails":"TimeoutError: [Errno Can't connect to all nodes of network] KILT","time":{"start":1669371632005,"stop":1669371633130,"duration":1125}},{"uid":"a0ca1c394f3c2130","reportUrl":"https://nova-wallet.github.io/test-runner/1946//#testresult/a0ca1c394f3c2130","status":"passed","time":{"start":1669119105238,"stop":1669119105238,"duration":0}}]},"tags":[]},"source":"1f7ae3c832eb1eeb.json","parameterValues":["<tests.utils.chain_model.Chain object at 0x7efcd43420d0>"]}